keyboards/takashicompany/minidivide_max/info.json

```diff
@@ -0,0 +1,156 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "miniDivide MAX",
+    "maintainer": "takashicompany",
+    "build": {
+        "lto": true
+    },
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to exist some functionality for them in a `minidivide_max.c` file using the `oled_task_kb()` function.

Reply:
I fixed.